<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UploaderDialog</class>
 <widget class="QDialog" name="UploaderDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>448</width>
    <height>208</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Upload screenshot</string>
  </property>
  <widget class="QPushButton" name="butClose">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>170</y>
     <width>102</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Close</string>
   </property>
  </widget>
  <widget class="QPushButton" name="butUpload">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>170</y>
     <width>102</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Ulpload</string>
   </property>
  </widget>
  <widget class="QLabel" name="shotLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>101</width>
     <height>101</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Panel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Sunken</enum>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="labResize">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>20</y>
     <width>241</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Resize uploaded image</string>
   </property>
  </widget>
  <widget class="QComboBox" name="cbxResize">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>50</y>
     <width>281</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
